tests/021.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+preg_replace() should untaint a string
+--SKIPIF--
+<?php if (!extension_loaded("taint")) print "skip"; ?>
+--INI--
+taint.enable=1
+--FILE--
+<?php
+class request {
+    function get() {
+        $a = "a" ."";
```

Comment:
this test is currently failing. after fiddling arround with it it seems that it depends on the used input-string and the pattern whether `preg_replace` untaints a string or not (I guess because it takes different paths in the `preg_replace` impl)